health_list/health_list_add.js

```diff
@@ -42,3 +46,22 @@ function afterImgHandler(e) {
     box.classList.add("active");
   };
 }
+
+confirmButton.addEventListener("click", () => {
+  let exerciseName = window.prompt("이름을 입력하세요");
+  const confirmed = window.confirm("운동을 추가하시겠습니까?");
+
+  const saveToLocalStorage = () => {
+    localStorage.setItem("saveData", JSON.stringify(data));
+  };
+  const data = {
+    name: exerciseName,
+    beforeStartImage: beforeImg.src,
+    afterStartVideo: afterImg.src,
+    progressPhrase: progressPhrase.value,
+    secondsPerCount: counterValue.textContent,
+  };
+  if (confirmed) {
+    saveToLocalStorage();
+  }
```

Comment:
어떠한 값을 변수에 담는 코드는 해당 변수가 여러 곳에서 사용될 일이 있을 때 쓰는게 좋습니다. 이용자의 confirm 여부는 분기 한곳에서만 사용되므로 다음과 같이 코드를 작성하는게 효율적이고 코드를 읽는데도 편합니다.

특정 코드 블록을 함수로 따로 빼는 것은 코드의 가독성을 늘려주는 좋은 습관이지만  saveToLocalStorage 는 단순히 한줄의 코드만을 대체할 뿐이므로 이 경우엔 함수를 따로 분리할 필요까지는 없습니다.

```suggestion
  const confirmed = window.confirm("운동을 추가하시겠습니까?");
    
  if (window.confirm("운동을 추가하시겠습니까?")) {
    const data = {
      name: exerciseName,
      beforeStartImage: beforeImg.src,
      afterStartVideo: afterImg.src,
      progressPhrase: progressPhrase.value,
      secondsPerCount: counterValue.textContent,
    };
    
    localStorage.setItem("saveData", JSON.stringify(data));
  }
```